pkg/controller/migmigration/quiesce.go

```diff
@@ -85,3 +86,67 @@ func (t *Task) scaleDownDeployments(client k8sclient.Client) error {
 
 	return nil
 }
+
+// Scales down all StatefulSets.
+func (t *Task) scaleStatefulSets(client k8sclient.Client) error {
+	zero := int32(0)
+	for _, ns := range t.PlanResources.MigPlan.Spec.Namespaces {
+		list := coreappsv1.StatefulSetList{}
+		options := k8sclient.InNamespace(ns)
+		err := client.List(
+			context.TODO(),
+			options,
+			&list)
+		if err != nil {
+			log.Trace(err)
+			return err
+		}
+		for _, deployment := range list.Items {
+			if deployment.Spec.Replicas == &zero {
+				continue
+			}
+			deployment.Spec.Replicas = &zero
+			err = client.Update(context.TODO(), &deployment)
+			if err != nil {
+				log.Trace(err)
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// Ensure scaled down pods have terminated.
+func (t *Task) ensureQuiescedPodsTerminated() (bool, error) {
```

Comment:
A comment describing the `bool` as essentially `terminated` or not would be great

Reply:
Done.